src/dpq2/conv/time.d

```diff
@@ -68,28 +85,55 @@ if( is( T == TimeStampWithoutTZ ) )
     );
 }
 
+/++
+    Structure to represent PostgreSQL's Timestamp without time zone type.
+    Conversions between timestamp without time zone and timestamp with time zone normally assume that
+    the timestamp without time zone value should be taken or given as timezone local time.
+    A different time zone can be specified for the conversion using AT TIME ZONE.
```

Comment:
Info about internal server conversions and some assumptions around it isn't theme of this comment and may confuse users.

My proposal is to replace this comment to short:

"Structure to represent PostgreSQL Timestamp with/without time zone."

Reply:
Patched